stryker.conf.js

```diff
@@ -23,7 +23,9 @@ module.exports = function(config) {
     },
     mutate: [
       "src/**/*.ts",
-      "!src/types/*d.ts"
+      "!src/types/*d.ts",
+      "!src/**/model/*.ts",
```

Comment:
After our conversation at code dojo this past week, you should still be mutating models like TimeZoneId. Please remove these exclusions.